app/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
```

Comment:
this could be a good chance to move from gradle to kts if you're up for it. not necessary for this PR though